activate.sh

```diff
@@ -7,27 +7,27 @@ autoenv_init() {
 
 	_mountpoint="$(df -P "${PWD}" | tail -n 1 | awk '{ for(i=6; i<NF; i++) printf "%s",$i OFS; if(NF) printf "%s",$NF; printf ORS}')"
```

Comment:
@Gnouc Indeed it was. I apologize for the sloppiness. This is just marginally shorter than your if statement 🤕 
`_mountpoint="$(df -P "${PWD}" | tail -n 1 | awk '{ for(i=6; i<NF; i++) printf "%s",$i OFS} END {print $NF}')"`